all_msgs/ros2interop/ros-types-all.xml

```diff
@@ -0,0 +1,1445 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+(c) 2019 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+RTI grants Licensee a license to use, modify, compile, and create derivative
+works of the Software.  Licensee has the right to distribute object form only
+for use with RTI products.  The Software is provided "as is", with no warranty
+of any type, including any warranty for fitness for any purpose. RTI is under
+no obligation to maintain or support the Software.  RTI shall not be liable for
+any incidental or consequential damages arising out of the use or inability to
+use the software.
+-->
+<types xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:noNamespaceSchemaLocation="http://community.rti.com/schema/6.0.0/rti_dds_topic_types.xsd">
+  <module name="std_msgs">
```

Comment:
Instead of "6.0.0" you can use "latest", that'll ensure you always use the most recent version of the XSD.

Reply:
Nice!   I didn't know about that.